assignments/nonlive/subscriptions/0511a7fe-771b-4ffa-9348-c59e9c4a87bd/assign.autotagging-application.json

```diff
@@ -0,0 +1,53 @@
+{
+  "id": "/subscriptions/0511a7fe-771b-4ffa-9348-c59e9c4a87bd/providers/Microsoft.Authorization/policyAssignments/HMCTSAutoTagging_Application_DTS-ARCHIVING-NONLIVE",
+  "identity": {
+    "type": "SystemAssigned"
+  },
+  "location": "uksouth",
+  "name": "HMCTSAutoTagging_Application_DTS-ARCHIVING-NONLIVE",
+  "properties": {
+    "description": "Policy to auto tag: application",
+    "displayName": "HMCTS Auto Tagging - Application - Archiving with Records Management NONLIVE (DTS-ARCHIVING-NONLIVE)",
+    "enforcementMode": "DoNotEnforce",
+    "metadata": {
+      "assignedBy": "https://github.com/hmcts/cpp-azure-policy"
+    },
+    "nonComplianceMessages": [],
+    "notScopes": [
+      "/subscriptions/0511a7fe-771b-4ffa-9348-c59e9c4a87bd/resourceGroups/AzureBackupRG_uksouth_1",
+      "/subscriptions/0511a7fe-771b-4ffa-9348-c59e9c4a87bd/resourceGroups/cloud-shell-storage-westeurope",
+      "/subscriptions/0511a7fe-771b-4ffa-9348-c59e9c4a87bd/resourceGroups/DefaultResourceGroup-SUK",
+      "/subscriptions/0511a7fe-771b-4ffa-9348-c59e9c4a87bd/resourceGroups/export-private-key-rg",
+      "/subscriptions/0511a7fe-771b-4ffa-9348-c59e9c4a87bd/resourceGroups/NetworkWatcherRG",
+      "/subscriptions/0511a7fe-771b-4ffa-9348-c59e9c4a87bd/resourceGroups/resource-graph-queries",
+      "/subscriptions/0511a7fe-771b-4ffa-9348-c59e9c4a87bd/resourceGroups/RG-AUDIT-DEV",
+      "/subscriptions/0511a7fe-771b-4ffa-9348-c59e9c4a87bd/resourceGroups/RG-DEV-IDM1-CORE-01",
+      "/subscriptions/0511a7fe-771b-4ffa-9348-c59e9c4a87bd/resourceGroups/RG-DEV-IDM1-OPS-01",
+      "/subscriptions/0511a7fe-771b-4ffa-9348-c59e9c4a87bd/resourceGroups/RG-idam-temp-backups-ea-14148",
+      "/subscriptions/0511a7fe-771b-4ffa-9348-c59e9c4a87bd/resourceGroups/RG-NFT-IDM1-CORE-01",
+      "/subscriptions/0511a7fe-771b-4ffa-9348-c59e9c4a87bd/resourceGroups/RG-NFT-IDM1-OPS-01",
+      "/subscriptions/0511a7fe-771b-4ffa-9348-c59e9c4a87bd/resourceGroups/RG-NFT-IDM2-CORE-01",
+      "/subscriptions/0511a7fe-771b-4ffa-9348-c59e9c4a87bd/resourceGroups/RG-SIT-IDM1-CORE-01",
+      "/subscriptions/0511a7fe-771b-4ffa-9348-c59e9c4a87bd/resourceGroups/RG-SIT-IDM1-OPS-01",
+      "/subscriptions/0511a7fe-771b-4ffa-9348-c59e9c4a87bd/resourceGroups/RG-SIT-IDM2-CORE-01",
+      "/subscriptions/0511a7fe-771b-4ffa-9348-c59e9c4a87bd/resourceGroup/RG-TMP-IDM1-01",
+      "/subscriptions/0511a7fe-771b-4ffa-9348-c59e9c4a87bd/resourceGroups/temp-rg",
+      "/subscriptions/0511a7fe-771b-4ffa-9348-c59e9c4a87bd/resourceGroups/temp_pip_rg"
+    ],
+    "parameters": {
+      "tagName": {
+        "value": "application"
+      },
+      "tagValue": {
+        "value": "archive-with-records-management"
```

Comment:
Change to IdAM (Check what existing IdAM resources have as the value for this tag)